### PR TITLE
Correct requirements

### PR DESCRIPTION
**Pull request by noah-de**
- The required version of Golang is 1.21.
- jq is not optional, it is a requirement.

[Addresses guacsec/guac#1068](https://github.com/guacsec/guac-docs/pull/77)

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -40,10 +40,10 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install)
+- [Go](https://go.dev/doc/install) (v1.21)
```
Comment (pxp928) [inline setup.md:43]: isn't the latest version currently v1.20.6? We could add "v1.20+"

**Review by pxp928 — COMMENTED** [review]
Thanks for the PR @noah-de! Quick comment else looks good.

File: setup.md (hunk 1 above)
Comment (noah-de) [reply setup.md:43]: When I ran it with v1.13 it failed with a variety of errors. 
Updating to the latest release told me that I needed v1.20.1 
Sorry for the typo. Shall I correct and issue another pull request?

Comment (pxp928) [reply setup.md:43]: You can just update this pull request with the correct one. I am not sure why the DCO fails for you. Might need to double check your git config

**Comment by noah-de** [comment]
Thank you @lumjjb  and @pxp928 
This PR should now be ready.

**Review by pxp928 — DISMISSED** [review]

File: setup.md [hunk 2]
```diff
@@ -40,10 +40,10 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install)
+- [Go](https://go.dev/doc/install) (v1.20.1)
```
Comment (pxp928) [inline setup.md:43]: hmm, actually I don't think it needs to be so specific. What error were you getting @noah-de?

Comment (pxp928) [reply setup.md:43]: I am currently on v1.20.3 and do not get any errors.

Comment (noah-de) [reply setup.md]: The machine that I was installing on had a much older version of Go, resulting in many errors.

I don't recall the specific errors and am on a different system today (no browser history).

@pxp928  I just changed it to what you had requested in https://github.com/guacsec/guac-docs/pull/78#discussion_r1266589384

**Review by pxp928 — APPROVED** [review]
LGTM!

**Pull request merged**